templates/zerver/help/add-bots-or-integrations.md

```diff
@@ -0,0 +1,48 @@
+# Add a Bot or Integration
+
+Bots and Integrations are features that are accesible to all members of an Organization.
+
+## To Add a Bot
+
+Bots can be created from the settings page:
+
+1. Click the cog (![cog](/static/images/help/cog.png)) in the upper right corner
+of the right sidebar.
+
+2. Select **Settings** from the dropdown menu that appears.
+
+    ![Settings Dropdown](/static/images/help/cog_dropdown_settings.png)
```

Comment:
I don't think you should capitalize the second word in the brackets, see other docs, e.g. https://github.com/zulip/zulip/commit/77d8056d5f60f12f596cc819ac3a21b6d8aa9139